addon/components/g-map-polyline.js

```diff
@@ -108,10 +109,27 @@ const GMapPolylineComponent = Ember.Component.extend({
     const { onClick } = this.attrs;
 
     if (typeOf(onClick) === 'function') {
-      onClick();
+      onClick(this);
     } else {
       this.sendAction('onClick');
     }
+  },
+
+  setOnDrag() {
+    const polyline = this.get('polyline');
+    if (isPresent(polyline)) {
+      polyline.addListener('dragend', () => this.sendOnDrag());
+    }
+  },
+
+  sendOnDrag() {
+    const { onDrag } = this.attrs;
+
+    if (typeOf(onDrag) === 'function') {
+      onDrag(this);
```

Comment:
how about sending `event` returned by event callback, instead of just `this`? I believe that event object should contain useful lat/lng data.

something like:
```
...
setOnDrag() {
  const polyline = this.get('polyline');
  if (isPresent(polyline)) {
    polyline.addListener('dragend', (e) => this.sendOnDrag(e));
  }
},

sendOnDrag(e) {
  const { onClick } = this.attrs;
  
  if (typeOf(onDrag) === 'function') {
    onDrag(e);
  else {
    this.sendAction('onDrag', e);
  }
},
```